www/js/lib/util.js

```diff
@@ -318,14 +241,11 @@ define([], function() {
         removeDuplicateStringsInSmallArray: removeDuplicateStringsInSmallArray,
         endsWith: endsWith,
         readIntegerFrom4Bytes: readIntegerFrom4Bytes,
-        readIntegerFrom2Bytes : readIntegerFrom2Bytes,
-        readFloatFrom4Bytes : readFloatFrom4Bytes,
-        uint8ArrayToHex : uint8ArrayToHex,
-        uint8ArrayToBase64 : uint8ArrayToBase64,
-        readFileSlice : readFileSlice,
+        readIntegerFrom2Bytes: readIntegerFrom2Bytes,
+        readFloatFrom4Bytes: readFloatFrom4Bytes,
+        uint8ArrayToHex: uint8ArrayToHex,
```

Comment:
A few of above functions are not used in the app any more. It's probably best to determine what to remove in a separate PR. I'll create an issue for it.

Reply:
See #843.